src/content-cli-analyze.ts

```diff
@@ -0,0 +1,43 @@
+import { ContextInitializer } from "./util/context-initializer";
+import { logger } from "./util/logger";
+import { ActionFlowCommand } from "./commands/action-flow.command";
+import commander = require("commander");
+
+type CommanderStatic = commander.CommanderStatic;
+
+class Analyze {
+    public static actionFlows(program: CommanderStatic): CommanderStatic {
+        program
+            .command("action-flows")
+            .description("Analyze Action Flows dependencies for a certain package")
+            .option("-p, --profile <profile>", "Profile which you want to use to analyze Action Flows")
+            .requiredOption("-i, --packageId <packageId>", "ID of the package from which you want to export Action Flows")
```

Comment:
I find the short hand notation very confusing, so would suggest to remove it.
```suggestion
            .requiredOption("--packageId <packageId>", "ID of the package from which you want to export Action Flows")
```

Reply:
Handled at [179f174](https://github.com/celonis/content-cli/pull/190/commits/179f174f5f59fbcf2817febdf0311a02e2974229)